pkg/cmd/run/shared/shared.go

```diff
@@ -97,7 +98,7 @@ type Run struct {
 	workflowName   string // cache column
 	WorkflowID     int64  `json:"workflow_id"`
 	Number         int64  `json:"run_number"`
-	Attempts       uint64 `json:"run_attempt"`
```

Comment:
> I'm genuinely surprised there is no other code that uses this field since it was added in https://github.com/cli/cli/pull/5945 🤯

I also went down this rabbit hole like gahhh. It's funny because it was even updated to use `uint64` instead of 8 bits because it was causing an unmarshal error even tho it was never useeeeeed.